qutebrowser/config/configfiles.py

```diff
@@ -814,7 +820,7 @@ def read_config_py(filename: str, raising: bool = False) -> None:
         raise configexc.ConfigFileErrors('config.py', api.errors)
 
 
-def read_autoconfig() -> None:
+def read_autoconfig(load_config: bool = True) -> None:
```

Comment:
I think this isn't needed anymore?

Reply:
Yes indeed, seems like I forgot to remove it